### ISSUE TITLE
LARBS not working properly.

### ISSUE THREAD

**Issue by ArchBtw8683** [LukeSmithxyz/LARBS#522]
I'm using Artix Linux, and I've used LARBS before. I don't know what I did wrong, or how I got LARBS working properly, but when I ran the LARBS script a month ago, it worked perfectly fine. I had to reinstall Artix, and did the same thing, and installed the LARBS script. This time the modules on the status bar, vim, and the caps lock key bind don't work properly. Vim isn't installed, the status bar doesn't show any modules, and the caps lock keybindings don't work, the caps lock key works.

**Comment by LukeSmithxyz** [comment]
Sounds like your user either doesn't have zsh as their user shell or the
zprofile isn't being called correctly. All problems here would be caused
by that.

If it isn't user error, I'll check on it when I can.

**Comment by ArchBtw8683** [comment]
Helo, Luke, thanks for getting back to me. I'll try and fix it manually from that. For some reason a lot of scripts I was using didn't work after the reinstall, some stopping at zsh.

**Comment by ArchBtw8683** [comment]
Zsh was the issue, installing zsh fixed it, and it's working perfectly now.

**Comment by ArchBtw8683** [comment]
What did you mean by the profile wasn't being called correctly?

At that time simply installing zsh and running zsh in the terminal, and running labs.sh through it worked.

I tried using my old zprofile and .zshrc file that was in my home partition, that didn't work neither.

I might've done something worng, but I did set zsh as my user shell.
$ echo $SHELL
/bin/zsh

**Comment by emrakyz** [comment]
@ArchBtw8683

Can you try these commands (AS ROOT) just to be sure (change 'yourusername' with your actual username):

```
pacman -S zsh
chown -R yourusername:yourusername /home/yourusername
chsh --shell /bin/zsh yourusername
```

Make sure you have this file as in the **"voidrice"** repo: `/home/yourusername/.config/shell/profile`
Also make sure you have this directory: `/home/yourusername/.config/zsh`

Then use the below command:
`ln -s /home/yourusername/.config/shell/profile /home/yourusername/.zprofile`

These steps make sure:
**1.** You install the zsh shell.
**2.** Your default user in the system have complete access to its home directory.
**3.** You change your default shell to zsh.
**4.** You have correct configuration files in correct places.
**5.** You create a symbolic link for ".zprofile" file.

**Comment by ArchBtw8683** [comment]
Hello, thanks for getting back to me.

I reinstalled Artix, kept my home partition, deleted the previous users, created a new user, and ran those commands after copying the content of Luke's Void rice in my .config folder;  I only copied the zsh and shell files.
I ran all of these commands as SU. My shell was again changed to zsh, but same issue, missing dwmblocks modules, and most of the features I use daily on LARBS isn't working.

I do have to say I'm using a live XFCE environment, not a fresh Artix installation. I can't simply reinstall Artix multiple times with a slow internet, it's just faster to use the installer that comes with the live environment.
 The live environment already comes with zsh installed, but it doesn't use zsh as its default shell.

I can try to run LARBS in a fresh installation of Artix, but I'll have to check if I can keep my /home partition.
